src/ingest_llm_as/db_models/raw_ingestion.py

```
@@ -0,0 +1,53 @@
+    content_type = Column(String(100), nullable=True)
+    raw_payload = Column(JSONB, nullable=False)
+    file_data = Column(LargeBinary, nullable=True)
+    raw_metadata = Column(JSONB, nullable=True)  # Renamed from 'metadata' (SQLAlchemy reserved keyword)
+    captured_at = Column(DateTime, nullable=False, default=datetime.utcnow)
+    processed = Column(Boolean, nullable=False, default=False)
+    processed_at = Column(DateTime, nullable=True)
+    processing_attempts = Column(Integer, nullable=False, default=0)
+    last_error = Column(Text, nullable=True)
+    created_at = Column(DateTime, nullable=False, default=datetime.utcnow)
+    
+    def __repr__(self):
```

Ruff (0.14.11)

Missing return type annotation for special method `__repr__`

Add return type annotation: `str`

(ANN204)